.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,8 +18,9 @@
 
 ## Release note
 
-Summarize the introduced changes in the code block below in one or a few sentences. The
-summary will be included in the next release notes automatically:
+We use [changelist](https://github.com/scientific-python/changelist) to
+compile each pull request into an item in our release notes. Please refer to
```

Comment:
```suggestion
list each pull request in the release notes. Please refer to
```

Reply:
"list" directly after "changelist" seems somewhat redundant.

What do you think about

> We use changelist to describe changes in each pull request in our release notes.

Reply:
Maybe it's just me and 'compile' is actually fine: I tend to think of the computing meaning *first* but really that's the *second* meaning of 'compile!'

Otherwise:

> We use changelist to log each pull request and its description in the release notes.

I find '*our* release notes' overly possessive :sweat_smile: